index.js

```diff
@@ -0,0 +1,36 @@
+$(document).ready(function() {
+	console.log('if i cant even get this right i dont know why im trying');
+
+
+	$('a').click(function(event) {
+		console.log('maybe if i get here ill feel a lil better');
+		event.preventDefault();
+	});
+
+	$('#posts .readmore a').click(function(){
+		console.log('what about here?');
+		$('.post p').removeClass('hide');
```

Comment:
You don't need to remove the class 'hide' because the JS `.show();` or `.hide();` overrides any CSS styles.